index.js

```diff
@@ -1,57 +1,32 @@
 const expect = require('chai').expect;
 
-const MajorColors = [
-    "white",
-    "red",
-    "black",
-    "yellow",
-    "violet"
-];
 
-const MinorColors = [
-    "blue",
-    "orange",
-    "green",
-    "brown",
-    "slate"
-];
+var get_Reference = require("./get_Color_Pair.js");
 
-function GetColorFromPairNumber(pairNumber) {
-    let colorPair = {};
-    const zeroBasedPairNumber = pairNumber - 1;
-    const majorColorIndex =
-        Math.floor(zeroBasedPairNumber / MinorColors.length);
-    colorPair.major = MajorColors[majorColorIndex];
-    colorPair.minor = MinorColors[zeroBasedPairNumber % MinorColors.length];
-    return colorPair
-}
-
-function GetPairNumberFromColors(majorColor, minorColor) {
-    let majorIndex = 0;
-    let minorIndex = 0;
-    for(majorIndex = 0; majorIndex < MajorColors.length; majorIndex++) {
-        if(MajorColors[majorIndex] == majorColor) {
-            break;
-        }
+function ToString()
+{   
+    console.log("Pin Number" + " " + " Major" + " " + "Minor");
+    
+    for(var i=1;i<=25;i++)
```

Comment:
Is it necessary to hard-code the 25-limit? Try to arrive at it from the array sizes, so that we don't have to reconcile everytime we add a color.